models/review.js

```diff
@@ -15,17 +15,19 @@ async function createReview(planId, userId, status) {
     let results = await pool.query(sql, [planId, userId, status, status, planId]);
     const reviewId = results[0][1].insertId;
 
+    // get the time that the review was created
     sql = "SELECT time FROM PlanReview WHERE reviewId=?;";
     results = await pool.query(sql, reviewId);
+    const time = results[0][0].time;
+
+    // send out notifications about the new status
+    planNotification(planId, userId, 2);
```

Comment:
`await`?

Reply:
So I want to get your opinion here. 

If the getLiveCourses function has no await. Should the notifications it spawns have awaits?
I think logically they should only have awaits if we will perform some action if the notifications fail to send.

This raises the question, should we perform an action if they fail to send?
Should we cancel the update if the notification at the beginning or end of the update fails?